coderd/notifications.go

```diff
@@ -110,13 +102,58 @@ func (api *API) putNotificationsSettings(rw http.ResponseWriter, r *http.Request
 	}
 
 	err = api.Database.UpsertNotificationsSettings(ctx, string(settingsJSON))
+	if err != nil {
+		if rbac.IsUnauthorizedError(err) {
+			httpapi.Forbidden(rw)
```

Comment:
suggestion: early return after here and then you can de-dent the else

Reply:
Good call 👍